fw/http_msg.c

```diff
@@ -545,6 +545,71 @@ tfw_http_msg_hdr_open(TfwHttpMsg *hm, unsigned char *hdr_start)
 	       hdr_start, *hdr_start, hdr->skb);
 }
 
+#define CALC_JA5H_SUMM(req, hash)		\
+do {						\
+	req->ja5h.summ *= 11;			\
```

Comment:
@EvgeniiMekhanik @Kutumov Guys, maybe we need to extract ja5 related stuff to the header file? Seems you are using same constants but with different names or even unnamed. Also it looks confusing when hashing functions are spread around.

Reply:
Yes fixed